### PR TITLE
Add a "density" kwarg to hist2d #11070

### LINKED ISSUE #11070: Add a "density" kwarg to hist2d
Recently `ax.hist` got a density keyword argument to conform to numpy kwargs - it would be good if `ax.hist2d` could have it's `normed` argument replaced with a `density` argument for API consistency.

### PR DESCRIPTION
**Pull request by thoo**
Fix #11070

### CODE REVIEW

**Comment by anntzer** [comment]
`normed` should be pushed all the way to the end to avoid backcompat issues.

**Comment by thoo** [comment]
@anntzer Could you elaborate more? Thanks.

**Comment by ImportanceOfBeingErnest** [comment]
What the above comment means is that the `normed` argument should appear last in the list of arguments, similar to how it's been done in #8993.

File: lib/matplotlib/axes/_axes.py [hunk 1]
```diff
@@ -6865,7 +6876,16 @@ def hist2d(self, x, y, bins=10, range=None, normed=False, weights=None,
           in effect to gamma correction) can be accomplished with
           `.colors.PowerNorm`.
         """
+        if density is not None and normed is not None:
+            raise ValueError("kwargs 'density' and 'normed' cannot be used "
+                             "simultaneously. "
```
Comment (timhoffm) [inline lib/matplotlib/axes/_axes.py]: This could be re-wrapped to save a line.

**Comment by thoo** [comment]
@timhoffm Thanks.

**Comment by timhoffm** [comment]
Unfortunately, our master branch has evolved since you branched your local copy. Therefore the pyplot code you could generate does not fit to the current master anymore. Please rebase your branch to the current upstream master (ask if you need advice how to do it).

**Comment by ImportanceOfBeingErnest** [comment]
Seems you have removed any commits from this PR. [Here is](https://matplotlib.org/devel/gitwash/development_workflow.html#rebasing-on-trunk) what the documentation tells about rebasing.

**Comment by thoo** [comment]
@timhoffm  @ImportanceOfBeingErnest  Thank for the reference. I went too far before I saw your comment. :) Here is my new attempt. Thanks for the help.

**Pull request closed without merging**